modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/dataset_map_sidebar.html

```diff
@@ -0,0 +1,21 @@
+{#
+    Displays a map showing the dataset extent on the dataset page sidebar
+
+    extent
+      GeoJSON representation of the geometry to draw (ie the value of the
+      'spatial' extra)
+
+      e.g.
+      {% set dataset_extent = h.get_pkg_dict_extra(c.pkg_dict, 'spatial', '') %}
+      {% if dataset_extent %}
+        {% snippet "spatial/snippets/dataset_map_sidebar.html", extent=dataset_extent %}
+      {% endif %}
+
+    #}
+
+    <section class="module module-narrow dataset-map">
+    <h2 class="module-heading">{{ _('Dataset extent') }}</h2>
+    <div class="module-content">
+      {% snippet "spatial/snippets/dataset_map_base.html", extent=extent %}
+    </div>
+    </section>
```

Comment:
This file could be named templates/spatial/snippets/dataset_map_sidebar.html and it should pick it up, if it doesn't plugin order might need to be changed.